src/components/Comments/WriteComment.js

```diff
@@ -151,13 +152,13 @@ export const styles = StyleSheet.create({
   columnContainer: {
     display: 'flex',
     flexDirection: 'column',
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
     flex: 1,
     width: '100%',
     position: 'absolute',
     bottom: 0,
     borderTopWidth: 1,
-    borderTopColor: COLORS.gray9,
+    borderTopColor: colors.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color and border top color of the `columnContainer` style have been updated. Ensure these color changes align with your design guidelines.
```diff
- backgroundColor: COLORS.white,
- borderTopColor: COLORS.gray9,
+ backgroundColor: colors.white,
+ borderTopColor: colors.gray1,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->